code/datums/components/chem_booster.dm

```diff
@@ -84,6 +91,7 @@
 /datum/component/chem_booster/proc/examine(datum/source, mob/user)
 	SIGNAL_HANDLER
 	to_chat(user, "<span class='notice'>The chemical system currently holds [resource_storage_current]u of green blood. Its' enhancement level is set to [boost_amount+1].</span>")
+	show_meds_beaker_contents()
```

Comment:
Will it not tochat null if it's on the floor when I observe it?

Reply:
changed to use the user